### PR TITLE
Python meetup blog post

### PR DESCRIPTION
**Pull request by COpenda**

### CODE REVIEW

File: _posts/Python-Meetup.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+author_id: 13
+status: draft
+type: post
+published: true
+meta:
+  _edit_last: '1'
+---
+Calling all pythonistas!
+
+We’re excited to be the official sponsors and hosts for the Nairobi Python meetup  going down this Saturday, 6th of August.
```
Comment (pld) [inline _posts/Python-Meetup.md]: missing "the" before "6th"

Comment (pld) [inline _posts/Python-Meetup.md]: missing "the" before "6th"

File: _posts/Python-Meetup.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+type: post
+published: true
+meta:
+  _edit_last: '1'
+---
+Calling all pythonistas!
+
+We’re excited to be the official sponsors and hosts for the Nairobi Python meetup  going down this Saturday, 6th of August. 
+
+This meetup will take place at Ona’s Nairobi office and will be graced by industry experts from BRCK,Echo Mobile, and Ona
```
Comment (pld) [inline _posts/Python-Meetup.md]: missing space after comma

**Comment by pld** [comment]
Let's add a link to the meetup posting, are there any images from there that we can use?

File: _posts/Python-Meetup.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+meta:
+  _edit_last: '1'
+---
+Calling all pythonistas!
+
+We’re excited to be the official sponsors and hosts for the Nairobi Python meetup  going down this Saturday, 6th of August. 
+
+This meetup will take place at Ona’s Nairobi office and will be graced by industry experts from BRCK,Echo Mobile, and Ona 
+
+Get ready to show off your intelligence, share best practices with other worthy pythonistas
```
Comment (pld) [inline _posts/Python-Meetup.md]: should be "and" instead of a comma

File: _posts/Python-Meetup.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+For more details about the event, please look at the Python Meetup
```
Comment (pld) [inline _posts/Python-Meetup.md]: needs to link to the meetup page

File: _posts/Python-Meetup.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+type: post
+published: true
+meta:
+  _edit_last: '1'
+---
+Calling all pythonistas!
+
+We’re excited to be the official sponsor and host for the first ever Python Expose meetup going down this Saturday, the 6th of August. 
+
+Python expose will be graced by presentations from industry experts speaking on:
```
Comment (pld) [inline _posts/Python-Meetup.md]: "expose" should be capitalized

File: _posts/Python-Meetup.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+* **Scaling Python/Django Applications** by *Dickson Ukang’a, Director of Engineering at Ona Kenya*
+* **Building Web Applications** by *Edwardo Biraghi, Software Engineer at BRCK*
+* **Python Open source** by *Simon Dunde, Cofounder of Echo Mobile*
+
+This meetup will be hosting mid to advanced level pythonistas but that doesn’t mean beginners can’t join in on the fun and learn from the best. 
+
+Python Expose will take place at Ona’s Nairobi office on 6th August from 9am to 1pm. Come ready to show off your intelligence and share best practices with other worthy pythonistas
```
Comment (pld) [inline _posts/Python-Meetup.md]: "the 6th of"

Comment (pld) [inline _posts/Python-Meetup.md]: end of sentence needs a period

File: _posts/Python-Meetup.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+For more details about the speakers or details on how to rsvp, please visit the [Python Expose meetup page](http://www.meetup.com/Python-Nairobi/) or tweet Frankie [@frankieonuonga](https://twitter.com/frankieonuonga)
```
Comment (pld) [inline _posts/Python-Meetup.md]: end of sentence needs a period

File: _posts/Python-Meetup.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+This meetup will be hosting mid to advanced level pythonistas but that doesn’t mean beginners can’t join in on the fun and learn from the best. 
+
+Python Expose will take place at Ona’s Nairobi office on 6th August from 9am to 1pm. Come ready to show off your intelligence and share best practices with other worthy pythonistas
+
+For more details about the speakers or details on how to rsvp, please visit the [Python Expose meetup page](http://www.meetup.com/Python-Nairobi/) or tweet Frankie [@frankieonuonga](https://twitter.com/frankieonuonga) 
+
+
+**When**: 6th August from 9am- 1pm EAT
+
+**Where**:
```
Comment (pld) [inline _posts/Python-Meetup.md]: can we embed a map?

**Comment by pld** [comment]
can we add any images? maybe the header here, http://www.meetup.com/Python-Nairobi/

**Comment by COpenda** [comment]
python meetup nairobi 1
python meetup nairobi 2

which of these two? The fisrt one is not of very good quality.

**Comment by COpenda** [comment]
@pld can we publish this if ok? Time is running out.

File: _posts/Python-Meetup.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Python Expose Meetup Nairobi
```
Comment (pld) [inline _posts/Python-Meetup.md]: I think this should be a sentance, like "Join us for the Python Expose Meetup in Nairobi" or something

File: _posts/Python-Meetup.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+![screen shot 2016-08-02 at 10 26 33 am](https://cloud.githubusercontent.com/assets/19772556/17320320/a7ca3a4a-589b-11e6-8edc-6b53571bd16e.png)
+
+Calling all pythonistas!
+
+<!-- more -->
+
+We’re excited to be the official sponsor and host for the first ever Python Expose meetup taking place this Saturday, the 6th of August. 
+
+Python expose will be graced by presentations from industry experts speaking on:
```
Comment (pld) [inline _posts/Python-Meetup.md]: casing

Comment (pld) [inline _posts/Python-Meetup.md]: this also probably sounds better with "The" in the front

File: _posts/Python-Meetup.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+* **Scaling Python/Django Applications** by *Dickson Ukang’a, Director of Engineering at Ona Kenya*
+* **Building Web Applications** by *[Edwardo Biraghi](https://twitter.com/sked1), Software Engineer at BRCK*
+* **Realtime Sensor Data Analysis using Echo-Sense** by *[Vitalis Shisoka](https://twitter.com/vitz_theguy), Cofounder of Echo Mobile*
```
Comment (pld) [inline _posts/Python-Meetup.md]: usually written "co-founder", that's what we use

File: _posts/Python-Meetup.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+Calling all pythonistas!
+
+<!-- more -->
+
+We’re excited to be the official sponsor and host for the first ever Python Expose meetup taking place this Saturday, the 6th of August. 
+
+Python expose will be graced by presentations from industry experts speaking on:
+
+* **Scaling Python/Django Applications** by *Dickson Ukang’a, Director of Engineering at Ona Kenya*
```
Comment (pld) [inline _posts/Python-Meetup.md]: I think we should use the title "Co-founder and Director of Engineering at Ona"

File: _posts/Python-Meetup.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+* **Building Web Applications** by *[Edwardo Biraghi](https://twitter.com/sked1), Software Engineer at BRCK*
+* **Realtime Sensor Data Analysis using Echo-Sense** by *[Vitalis Shisoka](https://twitter.com/vitz_theguy), Cofounder of Echo Mobile*
+
+This meetup will be hosting mid to advanced level pythonistas but that doesn’t mean beginners can’t join in on the fun and learn from the best. 
+
+Python Expose will take place at Ona’s Nairobi office on 6th August from 9am to 1pm. Come ready to show off your intelligence and share best practices with other worthy pythonistas.
```
Comment (pld) [inline _posts/Python-Meetup.md]: also "The"

Comment (pld) [inline _posts/Python-Meetup.md]: "the 6th"

File: _posts/Python-Meetup.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+For more details about the speakers or details on how to rsvp, please visit the [Python Expose meetup page](http://www.meetup.com/Python-Nairobi/) or tweet Frankie [@frankieonuonga](https://twitter.com/frankieonuonga).
```
Comment (pld) [inline _posts/Python-Meetup.md]: remove "details on how"

Comment (pld) [inline _posts/Python-Meetup.md]: capitalize RSVP

Comment (pld) [inline _posts/Python-Meetup.md]: I think also remove "Frankie" just leave the twitter handle

File: _posts/Python-Meetup.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+This meetup will be hosting mid to advanced level pythonistas but that doesn’t mean beginners can’t join in on the fun and learn from the best. 
+
+Python Expose will take place at Ona’s Nairobi office on 6th August from 9am to 1pm. Come ready to show off your intelligence and share best practices with other worthy pythonistas.
+
+For more details about the speakers or details on how to rsvp, please visit the [Python Expose meetup page](http://www.meetup.com/Python-Nairobi/) or tweet Frankie [@frankieonuonga](https://twitter.com/frankieonuonga). 
+
+
+**When**: 6th August from 9am- 1pm EAT.
+
+**Where**:
```
Comment (pld) [inline _posts/Python-Meetup.md]: can we embed a map?

**Pull request merged**